### PR TITLE
CHK-840: Add test for invoice address without zipcode

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

Adds test to cover the scenario of invoice address without zipcode in geolocation mode.

#### What problem is this solving?

Regression test.

#### How should this be manually tested?

Run `yarn test`

### CODE REVIEW

File: utils/invoice-actions.js [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
-  if (account !== ACCOUNT_NAMES.INVOICE) {
+  if (!INVOICE_ACCOUNTS.includes(account)) {
     return
   }
 
   waitLoad()
   cy.waitAndGet('.vtex-omnishipping-1-x-btnDelivery', 1000).click()
-  cy.focused().blur()
+
+  if (account === ACCOUNT_NAMES.INVOICE) {
```
Comment (lbebber) [inline utils/invoice-actions.js:17]: why this condition? (might warrant a lil comment)

**Review by lbebber — APPROVED** [review]
lgtm, just a suggestion

File: utils/invoice-actions.js (hunk 1 above)
Comment (lucasecdb) [reply utils/invoice-actions.js:17]: I don't know what this does, actually, I'll try to check the other invoice tests to see

Comment (lucasecdb) [reply utils/invoice-actions.js]: apparently this was so the click on the "go to payment" button worked correctly. It appears that it doesn't click properly if another element is already focused so I changed the function that clicks on the button to focus on it before attempting to click which should solve this

**Comment by lucasecdb** [comment]
the only failing test seems to be a bug in shipping-preview, but it is not related to this PR (although we should fix it to avoid false negatives)

**Review by silviasfon — APPROVED** [review]

**Review by lbebber — APPROVED** [review]

**Pull request merged**